py/bitbox02/bitbox02/communication/bitbox_api_protocol.py

```diff
@@ -704,9 +707,14 @@ def get_info(transport: TransportLayer) -> Tuple[str, Platform, Union[BitBox02Ed
         else:
             raise Exception("Unknown platform: {}".format(platform))
 
-        unlocked_byte = response[0]
+        unlocked_byte, response = response[0], response[1:]
         unlocked = {0x00: False, 0x01: True}[unlocked_byte]
-        return (version_str, platform, edition, unlocked)
+
+        initialized = None
+        if version_str >= semver.VersionInfo(9, 20, 0):
```

Comment:
Please test your changes - this check does not work, it crashes because `version_str` is not a valid version string.

Use parse_device_version(version_str) instead.

Reply:
right, thanks.